src/libYARP_sig/include/yarp/sig/Vector.h

```diff
@@ -457,16 +475,15 @@ class YARP_sig_API yarp::sig::Vector : public yarp::os::Portable
     void clear ()
     { storage.clear();}
 
-    ///////// Serialization methods
-    /*
+    /**
     * Read vector from a connection.
-    * return true iff a vector was read correctly
+    * @return true if a vector was read correctly
```

Comment:
iff had a meaning here, if we want to change this, we can discuss it, but this change does not belong in this PR, please keep the changes as small as possible

Reply:
This is the very first commit of this PR and everything after rely on it. I understand it is about Vector and not PC, but this context is the one giving a meaning to that commit.
A separated PR with only the change on the vector will be meaningless and confusing since it does not change the 'standard' behavior of the vector.

IMHO I believe it is better to have the change and it's usage together to provide a better understanding of why it was done.

Reply:
Sorry, I was referring just to the documentation change `iff` -> `if`, not to. Here "return true iff a vector was read correctly" doesn't mean "return true if a vector was read correctly" but "return true if a vector was read correctly, false otherwise.
Maybe the meaning of this *biconditional logical connective* (see [`iff` ](https://en.wikipedia.org/wiki/If_and_only_if), used btw in a lot of places (`git grep " iff " | wc -l` -> 114))  is not very clear and we can consider replacing them if we want to make it more explicit, but in this case this is changing 2 conditions to 1. The correct change would be:
```diff
-    * return true iff a vector was read correctly
+    * return true if and only if a vector was read correctly
```
or
```diff
-    * return true iff a vector was read correctly
+    * return true if a vector was read correctly, false otherwise
```
and in case, we should change it everywhere, not just here.

Reply:
can this method return true if the vector is not read correctly?